twrp.fstab

```diff
@@ -8,6 +8,9 @@
 /persist      ext4          /dev/block/bootdevice/by-name/persist        flags=fsflags=noatime,nosuid,nodev,barrier=1,noauto_da_alloc;mounttodecrypt
 /firmware     ext4          /dev/block/bootdevice/by-name/modem          flags=fsflags=ro,nosuid,nodev,barrier=0;mounttodecrypt
 /misc         emmc          /dev/block/bootdevice/by-name/misc
+/efs1         emmc          /dev/block/bootdevice/by-name/modemst1       flags=backup=1;wipeingui;display=EFS
+/efs2         emmc          /dev/block/bootdevice/by-name/modemst2       flags=backup=1;wipeingui;subpartitionof=/efs1
+/logo         emmc          /dev/block/bootdevice/by-name/logo           flags=display="logo";backup=1;flashimg=1
 
-/external_sd  auto          /dev/block/mmcblk1p1  /dev/block/mmcblk1     flags=display="MicroSD Card";storage;wipeingui;removable
-/usb_otg      auto          /dev/block/sda1       /dev/block/sda         flags=display="USB-OTG";storage;wipeingui;removable
\ No newline at end of file
+/external_sd  vfat          /dev/block/mmcblk1p1  /dev/block/mmcblk1     flags=display="external_sd";storage;wipeingui;removable;fsflags="utf8"

```

Comment:
It was a direct copy paste from the chinese twrp for addison. Which has SD working.